src/components/AddBook.css

```diff
@@ -41,3 +41,11 @@ form {
   width: 11.5rem;
   cursor: pointer;
 }
+
+#add:hover {
+  background-color: #1572b9;
+}
+
+#add:active {
+  background-color: #0290ff;
```

Comment:
- [ ] Please in order to follow the best practices of CSS you should not use `id` selectors  please try to change all the id selectors in your project to `Class` selectors 
![image](https://user-images.githubusercontent.com/61790587/172681483-823d4c9a-4ed8-434f-a574-aeef98c79546.png)